### PR TITLE
🌐 No more base as a language

### PR DESCRIPTION
**Pull request by rowanseymour**
... except in existing flows. It looks like we've been setting a default of `["eng"]` for flow languages for new orgs, but child orgs we're still being created with the flow languages of their parents, which we're sometimes empty.

### CODE REVIEW

File: temba/api/v2/serializers.py [hunk 1]
```diff
@@ -1617,32 +1617,33 @@ class WorkspaceReadSerializer(ReadSerializer):
...
     country = serializers.SerializerMethodField()
     languages = serializers.SerializerMethodField()
-    primary_language = serializers.SerializerMethodField()
     timezone = serializers.SerializerMethodField()
     date_style = serializers.SerializerMethodField()
-    credits = serializers.SerializerMethodField()
     anon = serializers.SerializerMethodField()
 
+    credits = serializers.SerializerMethodField()  # deprecated
+    primary_language = serializers.SerializerMethodField()  # deprecated
```
Comment (rowanseymour) [inline temba/api/v2/serializers.py:1625]: deprecated because it's just the first value in `languages`

File: temba/api/v2/serializers.py [hunk 2]
```diff
@@ -1651,9 +1652,9 @@ class Meta:
             "name",
             "country",
             "languages",
-            "primary_language",
             "timezone",
             "date_style",
-            "credits",
             "anon",
+            "credits",
+            "primary_language",
```
Comment (rowanseymour) [inline temba/api/v2/serializers.py:1659]: just punting deprecated fields to the end

File: temba/tests/crudl.py [hunk 3]
```diff
@@ -109,7 +111,7 @@ def as_user(user, allowed):
         as_user(viewer, allowed=allow_viewers)
         as_user(editor, allowed=allow_editors)
         as_user(agent, allowed=allow_agents)
-        as_user(org2_admin, allowed=allow_org2)
+        as_user(org2_admin, allowed=allow_org2, check_fields=False)
```
Comment (rowanseymour) [inline temba/tests/crudl.py:114]: because often forms have different fields for different orgs

**Comment by ericnewcomer** [comment]
Don't we need a flow migration too?

**Comment by rowanseymour** [comment]
Maybe but for now this is just trying to fix new flows

**Pull request merged**